VERSION

```diff
@@ -1 +1 @@
-7.7.0-SNAPSHOT
+7.7.0
```

Comment:
Should this change be merged to master or updated in the release branch?

Reply:
yeah, good catch - changing base to 7.7.x